src/app/legacy/containers/SocialEmbed/common/styles.js

```diff
@@ -25,7 +25,7 @@ const getWrapperHeightStyles = oEmbed => {
 };
 
 const Wrapper = styled.div`
-  margin-bottom: ${GEL_SPACING_TRPL};
+  margin: ${GEL_SPACING} 0;
```

Comment:
This will affect embeds in Articles as well. Is that required/expected by UX?

Reply:
yeah lol just realised that too ! Talking through it with UX now - can ignore this PR for now 🤦

Reply:
No probs! Just wanted to check 😄